python/src/main/python/interestingness/compute_interesting_diff.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
Was this file supposed to be committed?

Reply:
No, it wasn't - good catch.  It's for a future PR.  I'll do a push where I delete it, agreed?

Reply:
Sounds good!